rules/integrations/aws/privilege_escalation_sts_temp_creds_via_assume_role.toml

```diff
@@ -82,7 +136,7 @@ reference = "https://attack.mitre.org/tactics/TA0008/"
 
 [rule.new_terms]
 field = "new_terms_fields"
-value = ["user.id", "aws.cloudtrail.flattened.request_parameters.roleArn"]
+value = ["aws.cloudtrail.resources.arn", "aws.cloudtrail.user_identity.invoked_by"]
```

Comment:
`aws.cloudtrail.user_identity.invoked_by` isn't populated for event.types that are not `AWSService`, so we would get lots of false positives for `IAMUser` and `AssumedRole` types which are arguably the more suspicious identity types to track `AssumeRole` actions. We may need to create 2 distinct rules to do new_terms for both cases since we'd need to use `user.name` for the other.

Reply:
Created another rule to separate by AWS service vs IAMUser/Role. New Terms is then adjusted.

If by AWS Service --> Check commonality by ARN/Service
If by User/Role --> Check commonality by User/Role